### PR TITLE
Perfdash: category level

### PR DESCRIPTION
**Pull request by krzysied**
Adding category level selector.

### CODE REVIEW

File: perfdash/www/index.html [hunk 1]
```diff
@@ -11,17 +11,26 @@ <h1 class="md-display-2">Performance Dashboard</h1>
...
+            <md-select ng-model="controller.jobName" ng-change="controller.jobNameChanged()">
               <md-option ng-value="jobName" ng-repeat="jobName in controller.jobNames">
                 {{jobName}}
               </md-option>
             </md-select>
           </md-input-container>
         </div>
 	<div style="width: 200px; display: inline-block;">
+          <md-input-container>
+            <md-select placeholder="ComponentName" ng-model="controller.componentName"  ng-change="controller.componentNameChanged()">
```
Comment (shyamjvs) [inline perfdash/www/index.html]: Could you rename this to be `MetricCategory` instead of `ComponentName` to be more generic?

Comment (krzysied) [reply perfdash/www/index.html]: I will do this. However, I think `TestCategoryName` will be a better name - we are having `TestName` as the lowest level selector.

Comment (shyamjvs) [reply perfdash/www/index.html]: I'm not sure because this is not the category for the test, but category for the metric.

Comment (shyamjvs) [reply perfdash/www/index.html]: Sure, that makes sense.

File: perfdash/www/index.html [hunk 2]
```diff
@@ -23,14 +23,23 @@ <h1 class="md-display-2">Performance Dashboard</h1>
         </div>
 	<div style="width: 200px; display: inline-block;">
           <md-input-container>
-            <md-select placeholder="TestName" ng-model="controller.testName"  ng-change="controller.testNameChanged()">
-              <md-option ng-value="testName" ng-repeat="testName in controller.testNames">
-                {{testName}}
+            <md-select placeholder="CategoryName" ng-model="controller.categoryName"  ng-change="controller.categoryNameChanged()">
```
Comment (shyamjvs) [inline perfdash/www/index.html]: Sorry, I guess it wasn't clear earlier - I meant `MetricCategory` for this one (`CategoryName` doesn't say much by itself). So basically, JobName > MetricCategory > MetricName.

Comment (krzysied) [reply perfdash/www/index.html]: done.

**Pull request merged**